TestWebService/Sources/TestWebService/TestWebService.swift

```diff
@@ -41,6 +51,29 @@ struct TestWebService: Apodini.WebService {
     }
     
     var configuration: Configuration {
+        switch httpsConfig {
+        case .none:
+            HTTPConfiguration(
+                hostname: .init(address: "localhost", port: port),
```

Comment:
I think you can omit `"localhost"` and `"0.0.0.0"` as both arguments should have them as the default values. Also applies to the other cases.

Reply:
You're right that they could be removed. I kept them in here intentionally so that, if someone were to run the TestWebService, it'd be easier to see where the web service is running. Also, specifying just a port w/out an address did look kinda weird. I'll remove them if you want me to do so.

Reply:
I am fine with making it explicit if you want to do so but would then suggest to avoid specifying the string literal three times each. Maybe a static property of the `TestWebService` that is used in all switch cases would make sense to change the binding address and the hostname in a single location 👍

Reply:
I've changed it so that both `hostname` and `bindAddress` are now CLI options with a default value. that way they're defined in a central place, can also easily be changed, and additionally also will show up in the `--help` output.

Reply:
Nice, very cool! 🚀